src/copy.c

```diff
@@ -16,6 +16,103 @@ const u8 gUnk_080BAAF8[3][4] = {
         8,    8, 0x10, 0x20,
 };
 
+/*
+    This is a simple modify of sub_080095D0
+    It's weird that the if statement in this function gets ignored completely.
+    https://cexplore.karathan.at/z/rIt42N
+    Maybe just I made something wrong.
+*/
+
+#ifdef NONMATCHING
+
+u16 sub_08009238(u8 r0, u8 r1)
+{
+    u16 result;
```

Comment:
https://cexplore.karathan.at/z/8HqnwQ
This is nowhere near hard but you made so many mistakes. I believe you can make more effort.